docs/user-guide/services/index.md

```diff
@@ -147,11 +147,38 @@ Accessing a `Service` without a selector works the same as if it had selector.
 The traffic will be routed to endpoints defined by the user (`1.2.3.4:9376` in
 this example).
 
+A special case of service without selectors is an ExternalName service.
```

Comment:
"An ExternalName service is a special case of service that does not have selectors."